<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPostalCode</class>
 <widget class="QDialog" name="SearchPostalCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pesquisa de Cep Por Endereço...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/PNG/128x128/gnome-system-config-128x128.png</normaloff>:/images/PNG/128x128/gnome-system-config-128x128.png</iconset>
  </property>
  <widget class="QGroupBox" name="GBoxDataPostalCode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Dados</string>
   </property>
   <widget class="QLabel" name="LabelState">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Uf:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="CBoxState">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Selecionar o Estado da Localidade...</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCity">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cidade:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTypeAddress">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="CBoxTypeAddress">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Selecionar o Tipo de Endereço...</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelAddress">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Endereço:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditAddress">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>490</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar o Nome do Local/Endereço...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditAddressNumber">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar o Número para uma Pesquisa Apurada...</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelAddressNumber">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Número:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditPostalCode">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar o Cep/Cep localizado para o Endereço....</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LabelPostalCode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cep:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonConfirm">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Confirmar o uso do Cep Localizado...</string>
    </property>
    <property name="text">
     <string>Confirmar</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/128x128/agt_action_success-128.png</normaloff>:/images/PNG/128x128/agt_action_success-128.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonCleanForm">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Limpar os Campos do Formulário e Tabela de Visualização...</string>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/48x48/Rebuild-48x48.png</normaloff>:/images/PNG/48x48/Rebuild-48x48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonSearch">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Pesquisar o Cep Para os dados Informados....</string>
    </property>
    <property name="text">
     <string>Pesquisar</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/128x128/search-128.png</normaloff>:/images/PNG/128x128/search-128.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditCity">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>290</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar a Cidade do Endereço de Pesquisa...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelDistrict">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Bairro:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditDistrict">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar o Bairro/Localidade do Endereço de Pesquisa...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditComplements">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>340</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar o Complemento do Endereço de Pesquisa...</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelComplements">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Comp..:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GBoxLocale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>761</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Localidades:</string>
   </property>
   <widget class="QTableView" name="TableViewLocale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>741</width>
      <height>311</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Exibe os Logradouros encontrados para o Endereço...</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="TButtonExit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sair da Janela de Pesquisa de Cep...</string>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/PNG/48x48/Exithome-48x48.png</normaloff>:/images/PNG/48x48/Exithome-48x48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LabelImg">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>630</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images.qrc">:/images/PNG/logo-194x42.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
